open-company/open-company-web
# Fix search results load

Bug: search for a very old post, click it: 💥 post removed banner

To test:
- on mainline do a search
- click on show more results
- click on the last item (make sure that post is not in the first page of that section and AP)
- [x] 💥 post removed banner
- switch to this branch
- repeat the search
- click the same result
